src/graph/flow.rs

```diff
@@ -43,6 +43,10 @@ impl FlowGraph {
     ///
     /// Panics if the maximum flow is 2^63 or larger.
     pub fn dinic(&self, s: usize, t: usize) -> i64 {
```

Comment:
I'd like to avoid the extra function.
Feel free to modify both dinic() and mcf() to return tuples that include the flow.

Reply:
ok, changed to use 1 signature for both mcf & dinic